### DISCUSSION TITLE
the latest version 2022

### DISCUSSION THREAD

**Discussion by SoHailaNekHat** [Sharp-as-a-tech/chat-application#17] [Announcements]
## What's Changed
* create a license by @SoHailaNekHat in https://github.com/Sharp-as-a-tech/chat-application/pull/1
* Update ADD USER.jsp by @SabrinaHayat in https://github.com/Sharp-as-a-tech/chat-application/pull/7
* Update DELETE USER.jsp by @SabrinaHayat in https://github.com/Sharp-as-a-tech/chat-application/pull/8
* uploadservlet and the imagedownloader file updated by @SoHailaNekHat in https://github.com/Sharp-as-a-tech/chat-application/pull/9
* ChatClient file updated by @Yasnapopal in https://github.com/Sharp-as-a-tech/chat-application/pull/12
* the Add user and the delete user updated by @SabrinaHayat in https://github.com/Sharp-as-a-tech/chat-application/pull/13
* some changes were done by @SoHailaNekHat in https://github.com/Sharp-as-a-tech/chat-application/pull/15

## New Contributors
* @SoHailaNekHat made their first contribution in https://github.com/Sharp-as-a-tech/chat-application/pull/1
* @SabrinaHayat made their first contribution in https://github.com/Sharp-as-a-tech/chat-application/pull/7
* @Yasnapopal made their first contribution in https://github.com/Sharp-as-a-tech/chat-application/pull/12

**Full Changelog**: https://github.com/Sharp-as-a-tech/chat-application/commits/c1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Sharp-as-a-tech/chat-application/releases/tag/c1.0'>the latest version 2022</a>.</em>

**Comment by SabrinaHayat** [comment]
Salaam
It will be good for other people understanding!
 if we make a document for our this project in GitHub .
And then upload it here.

**Reply by Adela-aman** [reply]
yeah sure thing 
we will do that

**Comment by zahidasafi** [comment]
Nice👏

**Comment by Adela-aman** [comment]
great
